### DISCUSSION TITLE
这里想提供下几个代码的通用思路 各位也集思广益 看看有没有更好的代码逻辑

### DISCUSSION THREAD

**Discussion by FrundlesTian** [xjw580/Hearthstone-Script#183] [General]
目前的代码逻辑在我看来效率比较低，会包括解场，权重一类。 本人鄙见，代码应该改为下面几种：
1）纯速攻（猎人，瞎子，死骑）：总体思路全部打脸，其次是嘲讽随从
使用战吼随从--a.指向性指向嘲讽随从或英雄 b.发现性随机选择 c.只需要扔出去的战吼
法术--a.指向敌方英雄 b.(尽量不要塞) 发现随机选择
英雄技能打脸--区分不同职业即可
2）慢速流 （法师， 德鲁伊）：总体思路优先选择高费卡牌，思路先解场再打脸
法术--a.指向我方英雄 b.(尽量不要塞) 发现随机选择
发现选项--尽量选择高费

**Comment by sunchina2001** [comment]
我提供一个思路，战士，卡牌选取优先选取元素嘲讽和加buff（石母，洛瑟玛·塞隆，倒霉的炸药师傅，黑石摇滚，机甲爆王）等权重（5，4，1，6，2），然后带奔雷骏马（权重3），再带几张高费野兽方便奔雷骏马拉出，实测胜率50左右，遇到人机同行胜率70以上
